pkg/cmd/drtprod/scripts/tpcc_init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright 2024 The Cockroach Authors.
+#
+# Use of this software is governed by the CockroachDB Software License
+# included in the /LICENSE file.
+
+# This script sets up the tpcc import workload script in the workload node and starts the same in nohup
+# NOTE - This uses CLUSTER and WORKLOAD_CLUSTER environment variable, if not set the script fails
+
+if [ -z "${CLUSTER}" ]; then
+  echo "environment CLUSTER is not set"
+  exit 1
+fi
+
+if [ -z "${WORKLOAD_CLUSTER}" ]; then
+  echo "environment CLUSTER is not set"
+  exit 1
+fi
+
+# script is responsible for importing the tpcc database for workload
+roachprod ssh "${WORKLOAD_CLUSTER}":1 -- "tee tpcc_init.sh > /dev/null << EOF
+nohup ./workload fixtures import tpcc $(roachprod pgurl "${CLUSTER}":1) --db=cct_tpcc --checks=false $@ &
```

Comment:
We need to pass --warehouse flags as well.
And --partitions is optional if we want partitions.

Reply:
The warehouses flag is passed from the YAML